packages/@webex/plugin-meetings/src/locus-info/index.ts

```diff
@@ -35,6 +35,38 @@ import LocusDeltaParser from '../locus-info/parser';
  * @class LocusInfo
  */
 export default class LocusInfo extends EventsScope {
+  compareAndUpdateFlags: any;
+  emitChange: any;
+  locusParser: any;
+  meetingId: any;
+  parsedLocus: any;
+  updateMeeting: any;
+  webex: any;
+  aclUrl: any;
+  baseSequence: any;
+  created: any;
+  deltaParticipants: any;
+  identities: any;
+  membership: any;
+  participants: any;
+  participantsUrl: any;
+  replaces: any;
+  scheduledMeeting: any;
+  sequence: any;
+  controls: any;
+  conversationUrl: any;
+  embeddedApps: any;
+  fullState: any;
+  host: any;
+  info: any;
+  mediaShares: any;
+  replace: any;
+  url: any;
+
+  // JSTOTS: VERIFY DECLARED VARIABLES WHETHER ALL OF THEM BELONG HERE OR EXTENDED FROM PARENT CLASS
```

Comment:
can we do this "todo" in this PR? (same applies to the other 3 JSTOTS comments)

Reply:
Done